README.md

```diff
@@ -149,19 +191,12 @@ Mesh networking requires setting up links between each node in the cluster. This
 
```

Comment:
Is the tip command recommendation still valid?

Reply:
Its still valid for people building a cluster from docker run or with dynamic ip addresses.